pkg/gadgets/run/tracer/tracer.go

```diff
@@ -216,6 +216,9 @@ func (t *Tracer) createdByTracerMapMacro(tracerMapName string) bool {
 }
 
 func (t *Tracer) handleTracerMapDefinition(tracerMapName string) error {
+	heapMapName := "gadget_heap"
```

Comment:
Currently the only usage of `gadget_heap` is here, but for consistency this could be a `const` in `types/metadata.go`